src/shell.ts

```diff
@@ -50,7 +50,7 @@ export interface ShellResult {
 export type ShellHandler = (code: number, stdout: string, stderr: string) => void;
 
 function isWindows(): boolean {
-    return (process.platform === WINDOWS);
+    return (process.platform === WINDOWS) && !getUseWsl();
```

Comment:
I'm anxious this is going to break places where we use `isWindows()` to understand how to work with the _host_ filesystem, e.g. constructing file URIs for use with the `Diff` command.

Reply:
Tested that diff works correctly.